### DISCUSSION TITLE
LoadError: LoadError: UndefVarError: n_actions not defined

### DISCUSSION THREAD

**Discussion by longjianquan** [JuliaPOMDP/POMDPs.jl#366] [Debugging Help]
i run another pkg,it show above.i use a error branch?

**Comment by lassepe** [comment]
Hi,

`n_actions` has been removed in `v0.8` (https://github.com/JuliaPOMDP/POMDPs.jl/releases/tag/v0.8.0). Maybe the package that you are using did not set [compat bounds in the Project.toml](https://pkgdocs.julialang.org/v1/compatibility/) and thus is now using the new (incompatible) version of POMDPs.jl? If that is the case, a quick-fix would be to just add

```
[compat]
POMDPs = "< 0.8"
```

to your Project.toml.

If you want to use a later version of POMDPs.jl you would have to replace the uses of `n_actions(model)` with `length(actions(model))`.
